frontend/pages/profile/index.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from "react"
+
+import { Layout } from "../../shared-components"
+import { DashboardHeader } from "../../compositions"
+import ProfileNav from "../../compositions/profile-nav/profile-nav"
+import { ProfileMenu, UserDataType } from "../../models/profile"
+import {
+  ProfileInfo,
+  ProfileType,
+  SavedSearches,
+  SavedResults
+} from "../../compositions/profile-content"
+import styles from "./profile.module.css"
+
+// TODO: replace with API call
+import users from "../../models/__mocks__/users.json"
+
+export default function Profile() {
+  const [nav, setNav] = React.useState(ProfileMenu.USER_INFO)
+
+  const mockUser = users[0]
+
+  return (
+    <Layout>
+      <DashboardHeader />
+      <div className={styles.profileWrapper}>
```

Comment:
Nice! You might also consider supporting nested routes like /profile/info, /profile/saved-results etc. You'd [create a file like [[...path]].tsx](https://nextjs.org/docs/routing/dynamic-routes#optional-catch-all-routes) and re-export this component. Then useRouter will give you the path segments.

Reply:
thanks!